packages/rocketchat-webrtc/WebRTCClass.coffee

```diff
@@ -803,9 +803,11 @@ WebRTC = new class
 
 
 Meteor.startup ->
-	RocketChat.Notifications.onUser 'webrtc', (type, data) =>
-		if not data.room? then return
+	Tracker.autorun ->
```

Comment:
Ya, I'd agree. This logic seems better placed in `onUser`, but I was a bit hesitant as an outside contributor to go changing core methods without a firm understanding of the code base :smile: